### ISSUE TITLE
Nextcloud Crash

### ISSUE THREAD

**Issue by sdh91** [nextcloud/android#8210]
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.lang.NullPointerException: Attempt to read from field 'androidx.viewpager.widget.ViewPager com.owncloud.android.databinding.FileDetailsFragmentBinding.pager' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3774)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3950)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2289)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8456)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
Caused by: java.lang.NullPointerException: Attempt to read from field 'androidx.viewpager.widget.ViewPager com.owncloud.android.databinding.FileDetailsFragmentBinding.pager' on a null object reference
	at com.owncloud.android.ui.fragment.FileDetailFragment.getFileDetailSharingFragment(FileDetailFragment.java:156)
	at com.owncloud.android.ui.activity.FileActivity.getShareFileFragment(FileActivity.java:854)
	at com.owncloud.android.ui.activity.FileActivity.getFile(FileActivity.java:296)
	at com.owncloud.android.ui.activity.FileDisplayActivity.updateActionBarTitleAndHomeButton(FileDisplayActivity.java:1623)
	at com.owncloud.android.ui.activity.ToolbarActivity.setupToolbar(ToolbarActivity.java:100)
	at com.owncloud.android.ui.activity.ToolbarActivity.setupHomeSearchToolbarWithSortAndListButtons(ToolbarActivity.java:124)
	at com.owncloud.android.ui.activity.FileDisplayActivity.onCreate(FileDisplayActivity.java:262)
	at android.app.Activity.performCreate(Activity.java:8186)
	at android.app.Activity.performCreate(Activity.java:8170)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3747)
	... 11 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30150190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: p3q
Model: SM-G998U
Id: RP1A.200720.012
Product: p3qsqw

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: G998USQU1AUAJ

**Comment by tobiasKaminsky** [comment]
When does this happen?

**Comment by signal4siju** [comment]
Hi Tobias,

I am facing this issue too. It happens post failed connection to server> I have 2 url updated on the client one is internal address and the other is external link using NOIP and opening the port 443 in my router. SSL is not configured. I observed this error when I tested the connection after closing the port 443 in my router.  I tried opening the app to check connection, at first it tried fetching the data, then client gave me a prompt of the certificate which used to appear during the installation. I accepted the certificate and then this error struck which is reported by the initial user . Post the error, while opening the App, it just takes me to the error page, it doesn't gives me an option to select the other url unfortunately.
